Products/ZenCallHome/CallHomeStatus.py

```diff
@@ -76,10 +76,9 @@ def load_from_redis(self, k="CallHomeStatus"):
             log.warning(
                 "Exception trying to recive data from redis: {0}".format(e))
             self._redis_client = None
-            return
+            return dict()
```

Comment:
It looks like the `load_from_redis` method can now return _three_ different types of values: `None`, a `dict` object, and a string (or buffer) object.  A method or function should only be _one_ type or, at worst, two types as long as the second value type is `None`.